introduction/getting-started.qmd

```diff
@@ -28,7 +28,7 @@ We also recommend that you install and get to know the `tidyverse` set of packag
 install.packages("tidyverse")
 ```
 
-For introductions to R and the `tidyverse` we especially recommend the free resource [R for Data Science](https://r4ds.had.co.nz).
+For introductions to R and the `tidyverse` we especially recommend the free resource [*R for Data Science*](https://r4ds.had.co.nz).
```

Comment:
Just to confirm, in the book pdf the link text is italicized?

Reply:
In the pdf version it is not, however there is a note saying that the text should be italicized. 
![image](https://user-images.githubusercontent.com/65653573/236836361-704b6aab-4c5a-4252-b77d-eb660be263f5.png)

Reply:
ok great pls follow that

Reply:
Done!